dev.sparcles.SREngine.yml

```diff
@@ -0,0 +1,48 @@
+id: dev.sparcles.SREngine
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: srengine-run.sh
+
+build-options:
+  build-args:
+    - --share=network
```

Comment:
this is not allowed and will fail. Build MUST be offline.

Reply:
Removed these lines.